client/components/upgrades/google-apps/google-apps-dialog/index.jsx

```diff
@@ -47,27 +48,39 @@ const GoogleAppsDialog = React.createClass( {
 
 	render() {
 		const gapps = this.props.productsList && this.props.productsList.get().gapps;
-		let price = gapps && gapps.cost_display;
+		const price = gapps && gapps.cost_display;
 
 		// Gapps price is stored annually but we'd like to show a monthly price
-		price = price && price.replace( /(\d+\.?\d+)/, ( value ) => {
+		const monthlyPrice = price && price.replace( /(\d+\.?\d+)/, ( value ) => {
```

Comment:
Just a quick comment before I dig into this:
This (and line 59) might be suspectible to the same issue that I thought I fixed in https://github.com/Automattic/wp-calypso/pull/13527, but obvisously, missed this place 😞 Something like `\d+[.,]?\d+` should fix it 👍